packages/dmn-js-drd/src/features/drill-down/DrillDown.js

```diff
@@ -71,7 +72,7 @@ export default class DrillDown {
   addOverlay(element, className) {
     const html = domify(`
       <div class="drill-down-overlay">
-        <span class="${className}"></span>
+        <button class="${className}"></button>
```

Comment:
Since the span element doesn't capture ENTER key, I've changed this element to a Button which does capture it.
But also open to other ideas on how to bind the click event using span.

Reply:
We still need to fix styling here. Also, if the drilldown is not enabled (e.g. for DRD-only viewer), this should remain to be a `span` as it's not interactive.

Reply:
I added `type=button` because otherwise it's `type=submit` which is not correct.

Reply:
Yeah good point, about the styling added a comment here - https://github.com/bpmn-io/dmn-js/pull/802#discussion_r1407841798

I guess we want to add a blue background for the drill down button? 
Similar to what we had with the span element.

Reply:
> Also, if the drilldown is not enabled (e.g. for DRD-only viewer), this should remain to be a span as it's not interactive.

Is there some state we can check for "DRD-only viewer" mode? Then we could just add a conditional to render either a span or button:

```
let html;
    if (isDrdOnlyMode) {
      html = domify(`
      <div class="drill-down-overlay">
        <span type="button" class="${className}"></span>
      </div>
    `);
    } else {
      html = domify(`
      <div class="drill-down-overlay">
        <button type="button" class="${className}"></button>
      </div>
    `);
    }
```